Documentation/devicetree/bindings/gpio/gpio-ts7800v2.txt

```diff
@@ -0,0 +1,8 @@
+* embeddedTS TS-7800-V2 FPGA GPIO controller bindings
+
+This bindings describes the GPIO controller for embeddedTS's TS-7800-v2 FPGA
+core.
+
+Required properties:
+- compatible: Should be
+                "embeddedts,ts7800v2-gpio"
```

Comment:
This should be 'technologic,ts7800v2-gpio'  As discussed in engineering chat the other day, we're going to leave the vendor prefix alone upstream, which is currently technologic, due to how much of it is proliferated around the kernel. Putting it as technologic now would make us upstreaming this driver easier later.